js/main.js

```diff
@@ -34,6 +39,38 @@ var $extendedSpentSouvenirsInput = document.querySelector('#extended-spent-souve
+var $extendedTransportSpent = document.querySelector('#extended-transport-spent');
+var $extendedLodgingSpent = document.querySelector('#extended-lodging-spent');
+var $extendedFoodSpent = document.querySelector('#extended-food-spent');
+var $extendedActivitiesSpent = document.querySelector('#extended-activities-spent');
+var $extendedSouvenirsSpent = document.querySelector('#extended-souvenirs-spent');
+var $extendedReserveSpent = document.querySelector('#extended-reserve-spent');
+var $editDaySummaryLink = document.querySelector('#edit-day-summary-link');
+var $editExtendedSummaryLink = document.querySelector('#edit-extended-summary-link');
+var $editDayForm = document.querySelector('#edit-day-form');
+var $editExtendedForm = document.querySelector('#edit-extended-form');
+var $editDayBudgetTransportInput = document.querySelector('#edit-day-budget-transport-input');
+var $editDayBudgetFoodInput = document.querySelector('#edit-day-budget-food-input');
+var $editDayBudgetActivitiesInput = document.querySelector('#edit-day-budget-activities-input');
+var $editDayBudgetSouvenirsInput = document.querySelector('#edit-day-budget-souvenirs-input');
+var $editDayBudgetReserveInput = document.querySelector('#edit-day-budget-reserve-input');
+var $editDaySpentTransportInput = document.querySelector('#edit-day-spent-transport-input');
+var $editDaySpentFoodInput = document.querySelector('#edit-day-spent-food-input');
+var $editDaySpentActivitiesInput = document.querySelector('#edit-day-spent-activities-input');
+var $editDaySpentSouvenirsInput = document.querySelector('#edit-day-spent-souvenirs-input');
+var $editDaySpentReserveInput = document.querySelector('#edit-day-spent-reserve-input');
+var $editExtendedBudgetTransportInput = document.querySelector('#edit-extended-budget-transport-input');
+var $editExtendedBudgetLodgingInput = document.querySelector('#edit-extended-budget-lodging-input');
+var $editExtendedBudgetFoodInput = document.querySelector('#edit-extended-budget-food-input');
+var $editExtendedBudgetActivitiesInput = document.querySelector('#edit-extended-budget-activities-input');
+var $editExtendedBudgetSouvenirsInput = document.querySelector('#edit-extended-budget-souvenirs-input');
+var $editExtendedBudgetReserveInput = document.querySelector('#edit-extended-budget-reserve-input');
+var $editExtendedSpentTransportInput = document.querySelector('#edit-extended-spent-transport-input');
+var $editExtendedSpentLodgingInput = document.querySelector('#edit-extended-spent-lodging-input');
+var $editExtendedSpentFoodInput = document.querySelector('#edit-extended-spent-food-input');
+var $editExtendedSpentActivitiesInput = document.querySelector('#edit-extended-spent-activities-input');
+var $editExtendedSpentSouvenirsInput = document.querySelector('#edit-extended-spent-souvenirs-input');
+var $editExtendedSpentReserveInput = document.querySelector('#edit-extended-spent-reserve-input');
```

Comment:
Oh myyyyy 🤯 

Do all of these variables need to be global? It's okay if so, but if any of these global selectors could be relocated into specific functions they are used in, that would likely be for the best

Reply:
Gotcha, will relocate ones that do not need to be in global scope!